### ISSUE TITLE
Can't quit emulator without working game

### ISSUE THREAD

**Issue by LoungeKatt** [jpd002/Play-#425]
I'm not entirely sure why the exit button was removed from the menu. Games that give nothing but a black screen allow me to edit the theme, but not actually quit the emulator. Instead, I have to go back to the home screen, clear the app from memory, then launch it again.

**Comment by Thunder07** [comment]
on what platform?

**Comment by LoungeKatt** [comment]
I figured that would be obvious, unless you started adding themes to other platforms. Android. The exit menu only appears when the savestate menu is available, but that requires that the game loads.

**Comment by Thunder07** [comment]
It was, but I looked at the code, but found no logic in it that would cause this, thus I wanted to confirm.
anyway, the menu is loaded on the last line of OnCreate() method, so the only reason that line wont get called is if something crashed in the method earlier (which should crash the whole activity anyway).

however, I've noticed that after adding new elements, layouts or resources, that some of them wont load correctly, since they were not updated during build, and to solve this I either needed to have a clean build or remove Play/build_android/bin/ and  Play/build_android/gen/ directory.
